docs/UserGuide.md

```diff
@@ -621,15 +621,64 @@ You may use optional identifiers in conjunction with the minus(-) symbol to limi
 We understand that you might have commands that you frequently use. You may use shortcuts to abbreviate
 those commands to make using ClientBook more convenient.
 
+### <span style="color:#c9a500">Example Usage</span>
```

Comment:
The example usage could be a few lines to describe the process with image 3.
Image 1 and 2 can go into the respective command description.

Reply:
Thanks! I have made the changes by shifting the images into the respective command descriptions.